src/idl_gen_pybind.cpp

```diff
@@ -934,6 +980,12 @@ class PybindGenerator : public BaseGenerator {
     // The memoryview keeps the bytearray buffer alive.
     code_ += Indent() + "}, py::keep_alive<0, 2>());";
 
+    // operators.
+    if (opts_.gen_compare) {
+      code_ += Indent() + "{{BIND_VAR}}.def(py::self == py::self);";
```

Comment:
:exploding_head:

I am thinking of the machinery behind the `.def(py::self == py::self)` statement and I hope that at figure we won't do use anything like this. DSLs can be amazing, but learning C++ itself is hard enough.

But I have to admit that this does look really neat.